packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -264,6 +265,14 @@ class SemanticsProperties extends DiagnosticableTree {
     this.onMoveCursorBackwardByCharacter,
   });
 
+  /// If non-null, indicates that this subtree represents something that can be
+  /// in a disabled or enabled state.
```

Comment:
ultra-nit: 'enabled or disabled state' reads a bit better to me personally.

Reply:
Ok.